src/CameraHandler.cpp

```diff
@@ -25,8 +25,10 @@ CameraHandler::CameraHandler(std::string uniqueId) :
     try {
         init(uniqueId);
     } catch (...) {
-        release();
-        throw;
+        // if init failed we just make sure that mCamera is empty,
+        // but keep working as camerabe is mandatory service for DomA
```

Comment:
camera be doesn't know about Dom{A|U} or whatever: please avoid using those in comments

Reply:
Comment reworked.